src/main/java/com/k3ntako/HTTPServer/routes/SimpleRedirect.java

```diff
@@ -0,0 +1,15 @@
+package com.k3ntako.HTTPServer.routes;
+
+import com.k3ntako.HTTPServer.RequestInterface;
+import com.k3ntako.HTTPServer.Response;
+import com.k3ntako.HTTPServer.RouteInterface;
+
+public class SimpleRedirect implements RouteInterface {
+  @Override
+  public Response getResponse(RequestInterface request) {
+    var response = new Response();
+    response.setStatus(301);
+    response.addHeader("Location", "http://127.0.0.1:5000/simple_get");
```

Comment:
Totally good to make those feature tests pass! but once again, if thinking about real-life apps: do we always redirect to the same place?

Reply:
Consider having a `Redirect` class that can handle more cases